tests/regression/lp-1978005/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: apps can request not to be tracked
```

Comment:
Can you extend the existing app-awareness test instead? There's `tests/main/refresh-app-awareness`